Examples/rc-maestro/Project.swift

```diff
@@ -46,7 +46,7 @@ let project = Project(
                 configuration: "Debug",
                 executable: "Maestro-Debug",
                 options: .options(
-                    storeKitConfigurationPath: "rc-maestro/Resources/StoreKit/StoreKitConfigDefault.storekit"
+                    storeKitConfigurationPath: "rc-maestro/Resources/StoreKit/StoreKitConfiguration.storekit"
```

Comment:
This is the main change, right? I mean, the StoreKitConfiguration.storekit file is not synced while StoreKitConfigDefault.storekit is. Or am I missing something

Reply:
I learned that you can't make a synced file non-synced, so I had to create a new one and create new subscriptions.
That's why I tweaked the tests accordingly.

But yeah, the main change is generating the project with a different file

Reply:
Thanks for the explanation!